packages/components/src/Tooltip/__snapshots__/Tooltip.spec.ts.snap

```diff
@@ -0,0 +1,378 @@
+      class="mb-32 ml-32 button sizeLarge variantFlat colorBrand"
+      type="button"
+    >
+      Tooltip trigger
+    </button>
+  </div>
+  <div
+    data-tippy-root=""
+    id="tippy-5"
+    style="pointer-events: none; z-index: 9999; visibility: visible; position: absolute; left: 0px; top: 0px; margin: 0px; transform: translate(0px, 10px);"
+  >
+    <div
+      class="tippy-box tooltip variantLight"
+      data-animation="fade"
+      data-escaped=""
+      data-placement="bottom"
+      data-reference-hidden=""
+      data-state="visible"
+      role="tooltip"
+      style="max-width: 350px; transition-duration: 200ms;"
+      tabindex="-1"
+    >
+      <div
+        class="tippy-content"
+        data-state="visible"
+        style="transition-duration: 200ms;"
+      >
+        <div>
+          <span>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+             
+            <b>
+              Donec a erat eu augue consequat eleifend non vel sem.
+            </b>
+             Praesent efficitur mauris ac leo semper accumsan.
+          </span>
+        </div>
+      </div>
+      <div
+        class="tippy-arrow"
+        style="position: absolute; left: 0px; transform: translate(3px, 0px);"
+      />
+    </div>
+  </div>
+</body>
+`;
+
+exports[`<Tooltip /> DarkVariant story renders snapshot 1`] = `
+<body>
+  <div>
+    <button
+      aria-describedby="tippy-2"
+      class="mx-32 my-32 button sizeLarge variantFlat colorBrand"
+      type="button"
+    >
+      Tooltip trigger
+    </button>
+  </div>
+  <div
+    data-tippy-root=""
+    id="tippy-2"
+    style="pointer-events: none; z-index: 9999; visibility: visible; position: absolute; left: 0px; top: 0px; margin: 0px; transform: translate(10px, 0px);"
+  >
+    <div
+      class="tippy-box tooltip variantDark"
+      data-animation="fade"
+      data-escaped=""
+      data-placement="right"
+      data-reference-hidden=""
+      data-state="visible"
+      role="tooltip"
+      style="max-width: 350px; transition-duration: 200ms;"
+      tabindex="-1"
+    >
+      <div
+        class="tippy-content"
+        data-state="visible"
+        style="transition-duration: 200ms;"
+      >
+        <div>
+          <span>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+             
+            <b>
+              Donec a erat eu augue consequat eleifend non vel sem.
+            </b>
+             Praesent efficitur mauris ac leo semper accumsan.
+          </span>
+        </div>
+      </div>
+      <div
+        class="tippy-arrow"
+        style="position: absolute; top: 0px; transform: translate(0px, 3px);"
+      />
+    </div>
+  </div>
+</body>
+`;
+
+exports[`<Tooltip /> Interactive story renders snapshot 1`] = `
+<body>
+  <div>
+    <button
+      class="mx-32 my-32 button sizeLarge variantFlat colorBrand"
+      type="button"
+    >
+      Tooltip trigger
+    </button>
+  </div>
+</body>
+`;
```

Comment:
Oops, I just realized that the `Interactive` story isn't producing a useful snapshot because it's originally closed. I added it for manual interactive testing, so I feel like we should just tell this story to not make a snapshot. But I can also see the argument that we could use this story to verify that the tooltip does appear on hover or focus by adding a test that makes it appear on hover or focus and getting the snapshot. Thoughts?

Reply:
Hmmm, I'll mess around with the `play` function to see if I can get a meaningful snapshot while keeping it interactive, but if too much of a struggle, I can "unsnap" this